### DISCUSSION TITLE
How do I get the stats of a file?

### DISCUSSION THREAD

**Discussion by hubuhubugit** [igece/SoxSharp#24] [Q&A] [answered]
Like the stats result when running the command:
sox "filename" -n stat

I don't need to create new or modify files, I only want to get the "Volume adjustment" data from the stats.
I may be missing something but I didn't find a way to get this with SoxSharp.

**Comment by igece** [comment, marked as answer]
The `stat` effect is not currently implemented in SoxSharp. However, the library provides a way to call `sox` with these effects.
The code to perform your task is something like this:
```c#
using (var sox = new Sox("sox.exe"))
{
  sox.CustomEffects = "stat";
  sox.Process("filename");
}
```
